pandas/tests/series/methods/test_append.py

```diff
@@ -82,7 +83,12 @@ def test_append_dataframe_raises(self):
         with pytest.raises(TypeError, match=msg):
             df.A.append([df])
 
+    def test_append_raises_future_warning(self):
+        with tm.assert_produces_warning(FutureWarning):
```

Comment:
issue number

Reply:
Thanks! Where does the issue number go? Is there another example for this in the existing tests?

Reply:
Just below the definition, e.g.
```
def test_append_raises_future_warning(self):
    # GH#44539
```